credential/src/lib.rs

```diff
@@ -403,57 +408,57 @@ impl ProvideAwsCredentials for IamProvider {
 /// wrapped provider.  Each time the credentials are accessed, they are checked to see if
 /// they have expired, in which case they are retrieved from the wrapped provider again.
 pub struct BaseAutoRefreshingProvider<P, T> {
-	credentials_provider: P,
-	cached_credentials: T
+    credentials_provider: P,
+    cached_credentials: T
 }
 
 /// Threadsafe `AutoRefreshingProvider` that locks cached credentials with a `Mutex`
 pub type AutoRefreshingProviderSync<P> = BaseAutoRefreshingProvider<P, Mutex<AwsCredentials>>;
 
 impl <P: ProvideAwsCredentials> AutoRefreshingProviderSync<P> {
     pub fn with_mutex(provider: P) -> Result<AutoRefreshingProviderSync<P>, CredentialsError> {
-		let creds = try!(provider.credentials());
-		Ok(BaseAutoRefreshingProvider {
-			credentials_provider: provider,
-			cached_credentials: Mutex::new(creds)
-		})
-	}
+        let creds = try!(provider.credentials());
```

Comment:
Noting that a lot of these changes are whitespace and formatting, this would be a great opportunity to run this whole new sub-crate through rustfmt and keep it that way on subsequent commits (if that isn't the reason for these changes already)